libmodernize/main.py

```diff
@@ -96,6 +107,16 @@ def main(args=None):
             return 2
     if options.print_function:
         flags["print_function"] = True
+    if not _lib2to3_has_output_dir and (options.output_dir or options.add_suffix):
+        print("--output-dir and --add-suffix are not supported "
+              "with this version of Python.", file=sys.stderr)
+        return 2
+    # If we allowed these, the original files would be renamed to backup names
+    # but not replaced.
+    if options.output_dir and not options.nobackups:
+        parser.error("Can't use --output-dir/-o without -n.")
+    if options.add_suffix and not options.nobackups:
```

Comment:
These seem like strange checks. Does 2to3 do the same thing? Can we just make either of these options imply `nobackups`? The backups are only needed because it overwrites the source files - if you tell it not to do that, there's no need to think about backups.